service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RemoveOrderControllerSubmittedEvent.java

```diff
@@ -141,5 +199,176 @@ void shouldNotPublishPopulateStandardDirectionsEventIfASDOHasNotBeenRemoved() {
         postSubmittedEvent(callbackRequest);
 
         verifyNoMoreInteractions(coreCaseDataService);
+        checkThat(() -> verify(notificationClient, never())
+            .sendEmail(eq(SDO_REMOVAL_NOTIFICATION_TEMPLATE), any(), any(), any()));
+    }
+
+    @Test
+    void shouldNotSendEmailNotificationsIfABlankOrderIsRemoved() {
+        Element<GeneratedOrder> hiddenBlankOrder = element(buildBlankOrder());
+
+        CaseDetails caseDetails = CaseDetails.builder()
+            .data(Map.of(
```

Comment:
Would be nicer to build using case data instead of maps. You can do something like
```java
CaseDetails caseDetails = asCaseDetails(CaseData.builder().whatever().build());
```

Reply:
same for the other tests

Reply:
yeah, I agree. but `asCaseDetails()` is not setting the `Jurisdiction` and case type. I added a method to build the case data and updated all the tests.